docs/modules/ROOT/pages/cops_rspec.adoc

```diff
@@ -2868,11 +2870,54 @@ RSpec.describe Foo do
 end
 ----
 
+==== `EnforcedStyle: named_only`
+
+[source,ruby]
+----
+# bad
+RSpec.describe User do
+  subject(:user) { described_class.new }
+
+  it 'is valid' do
+    expect(subject.valid?).to be(true)
+  end
+end
+
+# good
+RSpec.describe Foo do
+  subject(:user) { described_class.new }
```

Comment:
The described class is Foo, maybe adjust the subject name to match?

Reply:
Hmm, it seem we describe a mixture of `Foo` and `User`, so I have made it consistently `describe User`. Hope that's OK.